sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -37,16 +37,24 @@
 """
 from __future__ import absolute_import
 
+from collections import namedtuple
+from random import shuffle
+
 import apache_beam as beam
+from apache_beam.io import iobase
+from apache_beam.io.range_trackers import LexicographicKeyRangeTracker
 from apache_beam.metrics import Metrics
 from apache_beam.transforms.display import DisplayDataItem
 
 try:
   from google.cloud.bigtable import Client
+  from google.cloud.bigtable.row_set import RowSet
+  from google.cloud.bigtable.row_set import RowRange
+  # from google.cloud.bigtable_v2.proto.bigtable_pb2 import SampleRowKeysResponse
```

Comment:
I'd also remove the commented import, as it's no longer relevant.

Reply:
yes, please.

Reply:
Done.